tests/example.js

```
@@ -0,0 +1,8 @@
+const unUsedVar = 'I am un-used'
+
+
+
+const myfunction= ()=> {
```

ESLint

[error] 'myfunction' is assigned a value but never used.

(no-unused-vars)